packages/multicast_dns/test/decode_test.dart

```diff
@@ -13,7 +13,6 @@ const int _kSrvHeaderSize = 6;
 void main() {
   testValidPackages();
   testBadPackages();
-  testNonUtf8Packages();
```

Comment:
nit: Maybe instead of removing this we, could update it to capture the behavior of inserting the UTF replacement character?

Reply:
Done.